### PR TITLE
Change some "info" log records to "warning" or "error"

### PR DESCRIPTION
**Pull request by vogdb**
Whenever something bad/unexpected happens it should be "warning" or "error".

E.g. this should be "warning":
```
{"level":"info","error":{"code":1,"message":"`ionice -c 2 -n 7 nice
```

cc @tomaszdurka

### CODE REVIEW

**Comment by vogdb** [comment]
@tomaszdurka please review

File: lib/job/manager.js [hunk 1]
```diff
@@ -56,19 +56,20 @@ JobManager.prototype.stop = function() {
...
     .then(function(jobDescription) {
       var jobId = path.basename(filePath, '.json');
       var handler = self.handlerRegistry.get(jobDescription['plugin'], jobDescription['event']);
-      var job = handler.instantiateJob(jobId, jobDescription['data']);
+      job = handler.instantiateJob(jobId, jobDescription['data']);
       return self._processor.processUntilSuccessful(job);
     })
     .then(function() {
-      serviceLocator.get('logger').info('Removing job file', {filepath: filePath});
+      serviceLocator.get('logger').info('Removing job file', {filepath: filePath, job: job ? job.serialize() : ''});
```
Comment (tomaszdurka) [inline lib/job/manager.js]: Don't pass empty string, probably better null.
Or rather `job && job.serialize()`.

**Comment by tomaszdurka** [comment]
This is quite inconsistent now.
We end up with:
```js
{error: 'error', job: {job: 'job-id'}}
```

We should rather:
```js
logger.debug('foo', _.extend({error: error}, job.serialize());
```

**Comment by tomaszdurka** [comment]
Method `serialize` of ours should be called differently. Something like `getContext` makes much more sense.

File: lib/job/manager.js (hunk 1 above)
Comment (tomaszdurka) [inline lib/job/manager.js]: With the new approach described in https://github.com/cargomedia/cm-janus/pull/229#issuecomment-205688721 let's rather:
```js
JobManager.prototype._processJobFile = function(filePath) {
  var self = this;
  var job;
  var getJobContext = function() {
    return job ? job.serialize() : null;
  };
  return self._readJobFile(filePath)
    .then(function(jobDescription) {
      var jobId = path.basename(filePath, '.json');
      var handler = self.handlerRegistry.get(jobDescription['plugin'], jobDescription['event']);
      job = handler.instantiateJob(jobId, jobDescription['data']);
      return self._processor.processUntilSuccessful(job);
    })
    .then(function() {
      serviceLocator.get('logger').info('Removing job file', _.extend({filepath: filePath}, getJobContext()));
      return fs.unlinkAsync(filePath);
    })
    .catch(function(error) {
      serviceLocator.get('logger').error('Processing job file failed', _.extend({error: error}, getJobContext()));
    });
};
```

**Comment by vogdb** [comment]
@tomaszdurka please look

File: lib/job/processor.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ JobProcessor.prototype._createJobWorkingDirectory = function(job) {
     fs.accessSync(dir, fs.W_OK);
   }
   catch (err) {
-    serviceLocator.get('logger').error('Job working directory could not be created');
+    serviceLocator.get('logger').error('Job working directory could not be created', job.serialize());
```
Comment (tomaszdurka) [inline lib/job/processor.js]: Shouldn't we add `err` too?

**Comment by tomaszdurka** [comment]
Rest looks good.

**Comment by vogdb** [comment]
@tomaszdurka please look

**Comment by tomaszdurka** [comment]
lgtm

**Pull request merged**